src/Libraries/Migrations/RevitNodes/Rectangle.cs

```diff
@@ -9,7 +9,7 @@ public class Rectangle : MigrationNode
         public static NodeMigrationData Migrate_0630_to_0700(NodeMigrationData data)
         {
             return MigrateToDsFunction(data, "ProtoGeometry.dll", "Rectangle.ByWidthHeight",
```

Comment:
It would be good to update the nickname as well, because Rectangle.ByWidthHeight node can't be searched in the library.